packages/react-topology/package.json

```diff
@@ -28,28 +28,28 @@
     "clean": "rimraf dist"
   },
   "dependencies": {
+    "@types/dagre": "0.7.42",
```

Comment:
seems like a workaround? what's the error/issue in org?

Reply:
The types don't really matter (we are inconsistent for deps like lodash and react-virtualized whether we move them to dev deps or normal deps).

What matters is that we install dagre and webcola which are used in `react-topology/src/layouts/index.ts`. Webpack will automatically try to require with an ESM import like `import { Whatever } from '@patternfly/react-topology';`.

Our options are to make them deps (what this PR does), or make them true peer deps and removing them from the main barrel file.

Reply:
`tslib` _is_ a dependency because `tsc` emits files that `require('tslib')`.

As for `@types` dependencies, I don't really think there is a standard. They can be deps (and automatically installed for consumers), dev deps, peer deps, or optional deps.